utilities/src/main/java/io/syndesis/qe/resource/impl/Syndesis.java

```diff
@@ -463,7 +488,7 @@ private void deploySyndesisViaOperator() {
                 serverFeatures.put("integrationStateCheckInterval", TestConfiguration.stateCheckInterval());
             }
             serverFeatures.put("integrationLimit", 5);
-
+            crJson.getJSONObject("spec").getJSONObject("addons").getJSONObject("todo").put("enabled", true);
```

Comment:
this shouldn't be needed since the todo is enabled by default in the `syndesis-minimal.yaml`, or am I missing something?

Reply:
It is not enabled in the [default-cr.yaml](https://github.com/syndesisio/fuse-online-install/blob/master/default-cr.yml) which will be using.

Reply:
right I realized that later and I thought I deleted this comment :grin: